contribs/gmf/examples/mobilemeasurelength.js

```diff
@@ -0,0 +1,112 @@
+goog.provide('gmf-mobilemeasurelength');
+
+goog.require('gmf.mobileMeasureLengthDirective');
+goog.require('ngeo.DecorateInteraction');
+goog.require('ngeo.MeasureEvent');
+goog.require('ngeo.btngroupDirective');
+goog.require('ngeo.interaction.MeasureLengthMobile');
+goog.require('ngeo.mapDirective');
+goog.require('ol.Map');
+goog.require('ol.Observable');
+goog.require('ol.Overlay');
+goog.require('ol.View');
+goog.require('ol.control.ScaleLine');
+goog.require('ol.layer.Tile');
+goog.require('ol.source.OSM');
+
+
+/** @const **/
+var app = {};
+
+
+/** @type {!angular.Module} **/
+app.module = angular.module('app', ['gmf']);
+
+
+/**
+ * App-specific directive wrapping the line measure tools. The directive's
+ * controller has a property "map" including a reference to the OpenLayers
+ * map.
+ *
+ * @return {angular.Directive} The Directive Definition Object.
+ */
+app.mobileMeasureLengthDirective = function() {
```

Comment:
Neither @jlap nor I understand what you mean.  Please, contact @jlap about this.